rotkehlchen/chain/ethereum/manager.py

```diff
@@ -393,27 +363,43 @@ def attempt_connect(
                         synchronized, msg = _is_synchronized(current_block, latest_block)
             except ValueError as e:
                 message = (
-                    f'Failed to connect to ethereum node {name} at endpoint '
+                    f'Failed to connect to ethereum node {node} at endpoint '
                     f'{ethrpc_endpoint} due to {str(e)}'
                 )
                 return False, message
 
             if not synchronized:
                 self.msg_aggregator.add_warning(
-                    f'We could not verify that ethereum node {name} is '
+                    f'We could not verify that ethereum node {node} is '
                     'synchronized with the ethereum mainnet. Balances and other queries '
                     'may be incorrect.',
                 )
 
-            log.info(f'Connected ethereum node {name} at {ethrpc_endpoint}')
-            self.web3_mapping[name] = web3
+            log.info(f'Connected ethereum node {node} at {ethrpc_endpoint}')
+            self.web3_mapping[node] = web3
             return True, ''
 
         # else
-        message = f'Failed to connect to ethereum node {name} at endpoint {ethrpc_endpoint}'
+        message = f'Failed to connect to ethereum node {node} at endpoint {ethrpc_endpoint}'
         log.warning(message)
         return False, message
 
+    def connect_to_multiple_nodes(self, nodes: Sequence[WeightedNode]) -> None:
+        for weighted_node in nodes:
+            if weighted_node.node == NodeName.ETHERSCAN:
+                logger.warning('Tried to connect to etherscan. Skipping')
```

Comment:
Will this always happen? If so don't log it.

Reply:
It doesn't happen always but if the user adds it it will happen. Have just removed the message